comps/llms/text-generation/vllm/vllm_arc.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+LLM_MODEL_ID="${LLM_MODEL_ID:=Intel/neural-chat-7b-v3-3}"
+
+source /opt/intel/oneapi/setvars.sh
+source /opt/intel/1ccl-wks/setvars.sh
+
+python -m ipex_llm.vllm.xpu.entrypoints.openai.api_server \
```

Comment:
IPEX-LLM is not yet an approved Intel product, so suggest holding on the relevant PRs for ARC support, unless the position is getting more clear.

Reply:
If ipex-llm not ready enough, do we have other alternate for ARC support currently? 
We can try to enable ARC with other alternates if we have.

Reply:
Official vLLM repo has supported ARC with OpenVINO backend.
Submitted new commit based on official vLLM repo